bindings_node/src/conversations.rs

```diff
@@ -81,6 +83,7 @@ impl From<GroupMembershipState> for XmtpGroupMembershipState {
       GroupMembershipState::Rejected => XmtpGroupMembershipState::Rejected,
       GroupMembershipState::Pending => XmtpGroupMembershipState::Pending,
       GroupMembershipState::Restored => XmtpGroupMembershipState::Restored,
+      GroupMembershipState::PendingRemove => XmtpGroupMembershipState::PendingRemove,
```

Comment:
add a todo for later to remove/refactor the GroupMembershipState, if it's really used meaningfully!